### DISCUSSION TITLE
How do I change Tesseract

### DISCUSSION THREAD

**Discussion by papa-pitufo** [SubtitleEdit/subtitleedit#5068] [General]
Sorry, but I don't see this question.

I have v3.6.1 and I can choose between Tesseract 3.0.2 or 5.0 Alpha, but if I see in Romaming/Subtitle edit I can see more Tesseract (4.0, 4.1.0, 4.1.1).

I would like to test my OCR with Tessearct 4.X but I don't known how to do it.

Thanks in advantage

Best regards

**Comment by niksedk** [comment]
Sorry, I see no need for Tesseract 4 as it's basically the same as Tesseract 5 beta... 

If you *really* want to test T4 you can rename your T4 folder to "Tesseract500.Alpha.20210506" and SE will use that when Tesseract 5 is selected.

You can find older Tesseract binaries here: https://digi.bib.uni-mannheim.de/tesseract/

**Comment by papa-pitufo** [comment]
Ok, thanks a lot
